packages/client/hmi-client/src/types/workflow.ts

```diff
@@ -51,10 +63,10 @@ export interface WorkflowEdge {
 	points: { x: number; y: number }[];
 
 	source: WorkflowNode['id'];
-	sourcePort: number;
+	sourcePortId: string;
```

Comment:
Ports will now be alias'd by a string so i can more easily find my inputs
am i understanding this correctly?

Reply:
I had to reread a bit my bad, ignore above